frontend/src/lib/Buttons.ts

```diff
@@ -654,6 +702,102 @@ export default class Buttons {
       callback(unsuccessfulIds, !unsuccessfulIds.length);
     }
   }
+
+  private _dialogActionHandlerMultiId(
+    selectedResourceAndParentIds: Map<string, string>[],
```

Comment:
nit: If I'm not mistaken, each map would have only one key:value pair. so you only need a tuple array: `[string, string][]`. And the possible value would be something like:
```typescript
[['pv-1', 'p-A'], ['pv-2', 'p-A'], ['pv-8', 'p-C']]`
```
Alternatively, if you use map, you only need `Map<string, string>` but not an array of map. And the possible value would be something like:
```typescript
{'pv-1' => 'p-A', 'pv-2' => 'p-A', 'pv-8' => 'p-C'}`
```

Reply:
Changed to the second options above.